doc/lxc.container.conf.sgml.in

```diff
@@ -430,7 +430,9 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
               network devices are usable in the container.  It also
               means that if both the container and host have upstart as
               init, 'halt' in a container (for instance) will shut down the
-              host.
+              host. Note that unprivileged containers do not work without this
+	      setting due to an inability to mount sysfs. An unsafe workaround
```

Comment:
Oh, one more: s/do not work without this setting/do not work with this setting/ :)

Reply:
Damn, I am never ever again doing a PR via the github web interface, all my mental safeguard's are not plugged into this. Fixed. thanks!